include/zephyr/mgmt/mcumgr/grp/fs_mgmt/fs_mgmt.h

```diff
@@ -22,6 +22,15 @@ extern "C" {
 #define FS_MGMT_ID_SUPPORTED_HASH_CHECKSUM	3
 #define FS_MGMT_ID_OPENED_FILE			4
 
+/**
+ * File operation.
+ */
+enum {
+	STATE_NO_UPLOAD_OR_DOWNLOAD = 0,
+	STATE_UPLOAD,
+	STATE_DOWNLOAD,
+};
```

Comment:
this needs a name for the enum starting with `fs_mgmt_` and `FS_MGMT_` prefix on values. Also as other comment states, should be encapsulated within a struct which includes the filename